otter/integration/config.sh-template

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+#
+# To run the integration tests, you'll need to have various environment
+# settings established.
+
+# This is the username used to authenticate against Identity.
+export AS_USERNAME=joe_user
+
+# This is the password used to authenticate against Identity.
+export AS_PASSWORD=joes_password
```

Comment:
No, it runs locally. Once it's on Github, it's too late. Furthermore, pre-commit hook means that it runs  *before* you commit.


In [the Makefile](https://github.com/rackerlabs/otter/blob/master/Makefile#L24-L28):
```
hooks:
	cp ${mkfile_dir}/scripts/config_check.py ${mkfile_dir}/.git/hooks
	echo "#!/bin/bash" > ${mkfile_dir}/.git/hooks/pre-commit
	echo "python .git/hooks/config_check.py" >> ${mkfile_dir}/.git/hooks/pre-commit
	chmod a+x ${mkfile_dir}/.git/hooks/pre-commit
```